### PR TITLE
Feature/erssup 63215 stu3

### PR DESCRIPTION
**Pull request by RajaGovindharaj16**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/components/stu3/examples/PasNotRespondingOperationOutcome.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (nhsd-david-wass) [inline specification/components/stu3/examples/PasNotRespondingOperationOutcome.json:1]: As for r4 the examples are copied from the sandbox during the build, so this directory should b added to .gitignore

**Comment by francisco-videira-nhs** [comment]
I wonder if it's worth rethinking having a `specification/components/common` directory.

If we do have one, then I think we should refactor/extract the most we can to the common directory.

For example, everything under` specification/components/r4/schemas/headers/request` is the same as and contained in `specification/components/stu3/schemas/headers/request`.

If we decide to not have one and favor keeping duplicated schemas in each separate dir, as there are some advantages to that, then `specification/components/common` would go to each separate fhir schema.

**Review by nhsd-david-wass — APPROVED** [review]
Looks good to me.
I agree with Francisco - if we are going to have a common area, then we should put as many common files as is logicakl in there ( particularly request/response headers and common error responses

**Review by nhsd-jack-wainwright — COMMENTED** [review]
Looks good to me 🙂

**Review by francisco-videira-nhs — APPROVED** [review]
LGTM 👍

**Review by francisco-videira-nhs — APPROVED** [review]

**Pull request merged**